src/app/shared/pages-scroll-cards/pages-scroll-cards.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="card-columns row">
+  <div *ngFor="let page of requestData" class="card p-4 text-center col-3">
+    <!-- content -->
+    <div class="card-block">
+      <h4 class="card-title"><a routerLink="{{page.url}}">{{page['title']}}</a></h4>
+      <p class="card-text">{{page['description']}}</p>
+    </div>
+    <!-- metadata -->
+    <!-- TODO: not working -->
+    <div *ngIf="showMeta == true" class="card-block">
```

Comment:
@asteroidice this is the line that doesn't work. showMeta is a variable in pages-scroll-cards.component.ts.

Reply:
Yeah, that's a fun one. I'm not completely sure why it's not working.